cmd/snap-assert/snap-assert.md

```diff
@@ -0,0 +1,36 @@
+# snap-assert: A Low-level Tool for Assertion Signing
+
+`snap-assert` is a low-level tool intended to be called by higher-level and more task specific tools when they need to create
+and sign specific assertions on behalf of the user:
+
+`snap-assert [OPTIONS] [<assert-type>] [<statement>]`
+
+## Keys and Signing
+
+`snap-assert` will use and invoke for signing a local GnuPG setup with its available keys. The GnuPG setup used will be the default one for the user (`~/.gnupg/`) unless `--gpp-homedir` is specified or the `GNUPGHOME` envvar is set.
+
+Keys for use need to be RSA and be at least 4096 bits long.
+
+## Selecting Keys and Identifying the Signer
+
+The signer identifier (the `authority-id` header value in the resulting assertion) can be specified with `--authority-id` while the key can be with `--key-id` giving a long key id. Otherwise the same can be achieved with`--account-key` to specify a file with the account key assertion for the intended key from which all the key and signer identifying information can be extracted.
```

Comment:
Maybe something like, "The signer identifier (the `authority-id` header value in the resulting assertion) can be specified with `--authority-id` together with `--key-id` to select the GnuPG key by long key id.", to better explain the two go together?

And I think s/account key assertion/`account-key` assertion.